keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -232,7 +232,7 @@ func (*BlinkingLightScorer) Score(state *CycleState, _ *v1.Pod, nodeName string)
 }
 ```
 
-However, the maximum count of blinking lights may be small compared to
+However, the maximum count of blinking lights may be big compared to
```

Comment:
"Small" and "big" both makes sense IMO. How about:

```suggestion
However, the maximum count of blinking lights may be small/big compared to
```

Reply:
@Huang-Wei  Thanks for quick replay,  the description is about `MaxNodeScore`, so if the real maximum count is small than `MaxNodeScore`, that's fine, no need to handle.